### ISSUE TITLE
Unexpected version after alpha release 

### ISSUE THREAD

**Issue by saig0** [camunda-community-hub/community-action-maven-release#2]
I want to build an alpha release of my project (e.g. `2.0.0-alpha1`). It works in general :heavy_check_mark: but it sets the version for the next release (i.e. `prepare for next development iteration`) to an unexpected value. 

Before:
```
<version>2.0.0-alpha1</version>
```

After:
```
<version>2.0.0-alpha2-SNAPSHOT</version>
```

Expected:
```
<version>2.0.0-SNAPSHOT</version>
```

See https://github.com/camunda-community-hub/zeeqs/commit/b2472c36b272bc579486255750e64d263a6e48ae

**Comment by celanthe** [comment]
Hi @saig0! Thank you for raising this issue. I had a look at [https://maven.apache.org/maven-release/maven-release-plugin/prepare-mojo.html](https://maven.apache.org/maven-release/maven-release-plugin/prepare-mojo.html) and I was wondering if there might be something there that would resolve this? 

There is also https://github.com/camunda-community-hub/maven-template which may be able to help as well. If neither of these has a solution, I can dig further into it and escalate to Infra if necessary!

**Comment by Langleu** [comment]
@saig0 the value is actually the expected value given by Maven, see given [documentation](https://maven.apache.org/guides/getting-started/index.html#what-is-a-snapshot-version).
Maven's default is to define the next development version as `x.y.(z+1)-SNAPSHOT`.

Due to KubeCon I won't find any time this week to have a look.

**Comment by celanthe** [comment]
Hi @saig0! Just wanted to follow back up on this--We have recently [released v1.0.4 of this extension](https://github.com/camunda-community-hub/community-action-maven-release/releases/), which merged in #14...Would this perhaps solve your issue? There was also #13 that may allow you to set different options for Snapshots. 

If neither of these new updates solve this, not to worry, I'll leave this issue open until we can resolve it! :)

**Comment by berndruecker** [comment]
Howdy @saig0 - I close this for now - please let me know if there is something still open - then we can have a look
